modules/EmailTemplates/EmailTemplateParser.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+class EmailTemplateParser
```

Comment:
This file is missing the license header and entrypoint check.  Please the the license header found in our coding standards https://suitecrm.com/wiki/index.php/Coding_Standards

Reply:
Fixed.